charts/catalog/templates/apiserver-cert-secret.yaml

```diff
@@ -9,8 +9,8 @@ metadata:
     heritage: "{{ .Release.Service }}"
 type: Opaque
 data:
-  tls.crt: {{ .Values.apiserver.tls.cert }}
-  tls.key: {{ .Values.apiserver.tls.key }}
+  tls.crt: {{ (.Files.Get .Values.apiserver.tls.certFileName) | b64enc }}
+  tls.key: {{ (.Files.Get .Values.apiserver.tls.keyFileName) | b64enc }}
```

Comment:
yes, it uses [sprig](https://github.com/masterminds/sprig) for template functions. relevant docs: https://masterminds.github.io/sprig/encoding.html